src/interface/interface.js

```diff
@@ -11,7 +11,7 @@ var Clipboard = require('clipboard')
 
 inherits(Interface, EventEmitter)
 
-function Interface () {
+function Interface() {
```

Comment:
My formatter is being a `${badWord}`. I can turn it off if you're attached to the spacing?